### ISSUE TITLE
feature request: rotate selection

### ISSUE THREAD

**Issue by sevenoffline** [fgebhart/mapa#52]
Is it a big deal to enable the user to rotate that selection rectangle?

**Comment by fgebhart** [comment]
TL;DR: yes!

Since mapa-streamlit is based on [streamlit-folium](https://github.com/randyzwitch/streamlit-folium), which itself is based on folium, my possibilities for what is possible with [the drawn rectangle](https://python-visualization.github.io/folium/modules.html#folium.vector_layers.Rectangle) are limited to the features these libraries offer.

Because of that I currently don't see an easy way of achieving the requested feature and would suggest to use blender or similar mesh editing tools as a workaround to e.g. cut a rotated selection from the output STL generated by mapa.

We can leave this open though, maybe somebody (including our future selves) knows a good solution to this.
